test/controllers/report/ReportingControllerSpec.scala

```diff
@@ -310,11 +373,60 @@ class ReportingControllerSpec extends UnitWithAppSpec {
   trait TestFixture extends TestFixtureBase {
     val frameworkId = "FastStream-2016"
 
-    def createAdjustmentsReport(frameworkId: String) = {
+    val SuccessfulAdjustmentReportResponse = Future.successful(
+      List(
+        AdjustmentReportItem("1", Some("11"), Some("John"), Some("Smith"), Some("Spiderman"), None, None, Some("Yes"),
+          Some(ApplicationStatus.SUBMITTED), Some("Need help for online tests"), Some("Need help at the venue"),
+          Some("Yes"), Some("A wooden leg"),
+          Some(Adjustments(Some(List("etrayTimeExtension")),Some(true),Some(AdjustmentDetail(Some(55),None,None)),None)),None),
+        AdjustmentReportItem("2", Some("22"), Some("Jones"), Some("Batman"), None, None, None, None,
+          Some(ApplicationStatus.PHASE1_TESTS), None, Some("Need help at the venue"), None, None,
+          Some(Adjustments(Some(List("etrayTimeExtension")),Some(true),Some(AdjustmentDetail(Some(55),None,None)),None)),None),
+        AdjustmentReportItem("3", Some("33"), Some("Kathrine"), Some("Jones"), Some("Supergirl"), None, None, None,
+          Some(ApplicationStatus.PHASE1_TESTS_PASSED), Some("Need help for online tests"), None,
+          Some("Yes"), Some("A glass eye"),
+          Some(Adjustments(Some(List("etrayTimeExtension")),Some(true),Some(AdjustmentDetail(Some(55),None,None)),None)),None)
+      )
+    )
+
+    val SuccessfulProgressReportResponse = Future.successful(
+      List(
+        CandidateProgressReportItem("user1", "app1", Some("submitted"),
+          List(SchemeType.DiplomaticService, SchemeType.GovernmentOperationalResearchService), Some("Yes"),
+          Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("1234567"), None, None),
+        CandidateProgressReportItem("user2", "app2", Some("registered"),
+          List(SchemeType.DiplomaticService, SchemeType.GovernmentOperationalResearchService), Some("Yes"),
+          Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("1234567"), None, Some("Faststream")),
+        CandidateProgressReportItem("user3", "app3", Some("submitted"),
+          List(SchemeType.DiplomaticService, SchemeType.GovernmentOperationalResearchService), Some("Yes"),
+          Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("1234567"), None, Some("Edip")),
+        CandidateProgressReportItem("user4", "app4", Some("submitted"),
+          List(SchemeType.DiplomaticService, SchemeType.GovernmentOperationalResearchService), Some("Yes"),
+          Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("No"), Some("1234567"), None, None)
+      )
+    )
+
+    val SuccessfulFindAllPostCodeResponse = Future.successful(
+      Map(
+        "user1" -> "EH2 4AD",
```

Comment:
We shouldn't be putting real data in the unit tests. I appreciate we need a matching prefix though. Postcodes such as EH9 9ZZ don't exist yet (and probably never will)